pynest/nest/lib/hl_api_subnets.py

```diff
@@ -49,16 +47,9 @@ def PrintNetwork(depth=1, subnet=None):
     """
 
     if subnet is None:
-        # Turn off deprecation warning on Python and SLI level as users
-        # shouldn't change implementation of PrintNetwork, it is done by the
-        # developers.
-        deprecation_bool, verbosity_level = turn_off_deprecation_warning(
-            'CurrentSubnet')
-        subnet = CurrentSubnet()
-        # Need to reset the deprecation warning to its old value
-        turn_on_deprecation_warning('CurrentSubnet', deprecation_bool,
-                                    verbosity_level)
-
+        # Avoid confusing user by deprecation warning
+        with SuppressedDeprecationWarning('CurrentSubnet'):
```

Comment:
Note that if you indeed chose to make this not available to users (see comment above), you'll need to import it explicitly